x-pack/plugins/fleet/server/integration_tests/router.test.ts

```diff
@@ -19,7 +19,6 @@ import * as kbnTestServer from '../../../../../src/test_utils/kbn_server';
 function createXPackRoot(config: {} = {}) {
   return kbnTestServer.createRoot({
     plugins: {
-      scanDirs: [],
```

Comment:
@elastic/fleet I just removed an usage from a fully commented test, so I'll take the liberty to merge on `core`'s approval if your review didn't land yet.